src/Form/input-classifiers.js

```diff
@@ -24,7 +25,13 @@ const getExplicitLabelsText = (el) => {
  */
 const getRelatedText = (el, form) => {
     const container = getLargestMeaningfulContainer(el, form)
-    return container.textContent
+
+    // If there is no meaningful container return empty string
+    if (container === el || container.nodeName === 'SELECT') return ''
+
+    // If the container has a select element, remove its contents to avoid noise
+    const noisyText = container.querySelector('select')?.textContent || ''
+    return container.textContent?.replace(noisyText, '')
```

Comment:
`select` elements are not relevant here and they usually contain a lot of noise that can match random stuff (for example, the list of countries in a select element can match some of the identities regexes). Here I try to remove the noise.